<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI::EditEntityDialog</class>
 <widget class="QDialog" name="GUI::EditEntityDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit entity</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="leName">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>3</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>Enter entity name here...</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lstMembers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
           <horstretch>3</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string notr="true">Properties</string>
          </property>
          <property name="icon">
           <iconset resource="main.qrc">
            <normaloff>:/icons/pic/icon_properties.png</normaloff>:/icons/pic/icon_properties.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Methods</string>
          </property>
          <property name="icon">
           <iconset resource="main.qrc">
            <normaloff>:/icons/pic/icon_method.png</normaloff>:/icons/pic/icon_method.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Fields</string>
          </property>
          <property name="icon">
           <iconset resource="main.qrc">
            <normaloff>:/icons/pic/icon_field.png</normaloff>:/icons/pic/icon_field.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">Elements</string>
          </property>
          <property name="icon">
           <iconset resource="main.qrc">
            <normaloff>:/icons/pic/icon_elements.png</normaloff>:/icons/pic/icon_elements.png</iconset>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QComboBox" name="cbType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QComboBox" name="cbScopes">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbCreateScope">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>29</width>
               <height>29</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Click to create new scope</string>
             </property>
             <property name="text">
              <string>+</string>
             </property>
             <property name="shortcut">
              <string>Ctrl+N</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="viewMembers">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>7</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>32</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>26</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>26</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pbNewComponent">
           <property name="toolTip">
            <string>Click to add a new component</string>
           </property>
           <property name="text">
            <string>&amp;Add new class component</string>
           </property>
           <property name="icon">
            <iconset resource="main.qrc">
             <normaloff>:/icons/pic/icon_component_plus.png</normaloff>:/icons/pic/icon_component_plus.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+Shift+N</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbEdit">
           <property name="toolTip">
            <string>Click to edit selected component</string>
           </property>
           <property name="text">
            <string>&amp;Edit</string>
           </property>
           <property name="icon">
            <iconset resource="main.qrc">
             <normaloff>:/icons/pic/icon_component_edit.png</normaloff>:/icons/pic/icon_component_edit.png</iconset>
           </property>
           <property name="shortcut">
            <string>Ctrl+E</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDelete">
           <property name="toolTip">
            <string>Click to remove selected components</string>
           </property>
           <property name="text">
            <string>&amp;Delete</string>
           </property>
           <property name="icon">
            <iconset resource="main.qrc">
             <normaloff>:/icons/pic/icon_component_delete.png</normaloff>:/icons/pic/icon_component_delete.png</iconset>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QToolButton" name="tbUndo">
         <property name="toolTip">
          <string>Press to undo last action</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/icons/pic/icon_undo.png</normaloff>:/icons/pic/icon_undo.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbRedo">
         <property name="toolTip">
          <string>Press to redo last action</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="main.qrc">
           <normaloff>:/icons/pic/icon_redo.png</normaloff>:/icons/pic/icon_redo.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Shift+Z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbReject">
       <property name="toolTip">
        <string>Click to forget all changes</string>
       </property>
       <property name="text">
        <string>&amp;Reject</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/icons/pic/icon_reject.png</normaloff>:/icons/pic/icon_reject.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAccept">
       <property name="toolTip">
        <string>Click to save all changes</string>
       </property>
       <property name="text">
        <string>&amp;Accept</string>
       </property>
       <property name="icon">
        <iconset resource="main.qrc">
         <normaloff>:/icons/pic/icon_accept_edit_dialog.png</normaloff>:/icons/pic/icon_accept_edit_dialog.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
